src/app.ts

```diff
@@ -912,8 +912,8 @@ const awsx = (): void => {
       },
     })
     .command({
-      command: 'add-key-expiry [profile] [expiry-period]',
-      describe: "Add expiry to your profile's secret key",
+      command: 'add-key-max-age [profile] [max-age]',
+      describe: 'Add max age of profile access key in days',
```

Comment:
can we put an example of the command here to make it easy for the user to copy / paste?